### ISSUE TITLE
Issues with the Stanford demo

### ISSUE THREAD

**Issue by amineremache** [chrischoy/SpatioTemporalSegmentation#32]
Hello,
I tried to run the Stanford demo, and I got multiple issues:
1-  When you run the ``python -m demo.stanford`` command for the first time (you don't have the ``.ply`` file yet) it should run the download function since the ``file_name`` doesn't match with any local file. 
![image](https://user-images.githubusercontent.com/19911661/78586189-61f3e980-783b-11ea-9ffd-cc935bcbeeb6.png)
But then, you get this error:
![image](https://user-images.githubusercontent.com/19911661/78586401-ada69300-783b-11ea-8202-1817045a5fcb.png)
Because the defined ``file_name`` in the start of the demo file doesn't match the name of the file on the server-side if you follow the static part of the ![link](http://cvgl.stanford.edu/data2/minkowskiengine/) (conferenceRoom_1.ply vs 1.ply).
![image](https://user-images.githubusercontent.com/19911661/78586513-d464c980-783b-11ea-90fe-d1f442b6a3b9.png)

![image](https://user-images.githubusercontent.com/19911661/78586731-24dc2700-783c-11ea-8247-037b2191d44d.png)

2- When you change the name on the code to ``1.ply``, it downloads the file, but from what I understood, it is not the good file.
Because then I got this error:
![image](https://user-images.githubusercontent.com/19911661/78587009-a2a03280-783c-11ea-8056-40ac438e4368.png)

Which is caused by this line:
![image](https://user-images.githubusercontent.com/19911661/78587053-b64b9900-783c-11ea-96d4-b968bf4be546.png)

And when you see the structure of the data here:
![image](https://user-images.githubusercontent.com/19911661/78587196-e4c97400-783c-11ea-9afd-abda05cfc031.png)

It really has no "label" property, and I think the file is from ScanNet because when you visualize it you get this:
![image](https://user-images.githubusercontent.com/19911661/78587448-3ffb6680-783d-11ea-82f4-2676882de63a.png)

Sorry if this is not the proper way to write an issue, it is my first one, and I tried to be the most explicit I could.

**Comment by ghost** [comment]
Hi,

That's because `1.ply` is from Scannet dataset & doesn't have the label attribute. What you need is a test file from S3DIS dataset. 

Another hack would be to just pass `torch.ones(SHAPE)` as labels variable in `stanford.py`, it will run.

**Comment by amineremache** [comment]
> Hi,
> 
> That's because `1.ply` is from Scannet dataset & doesn't have the label attribute. What you need is a test file from S3DIS dataset.
> 
> Another hack would be to just pass `torch.ones(SHAPE)` as labels variable in `stanford.py`, it will run.

Yeah, I was ( still ? xD ) a "noob" back then, it was my first few days with this code base, but things are good now, thanks for your answer.
